conans/client/new_ci.py

```diff
@@ -195,11 +249,18 @@ def ci_get_files(name, version, user, channel, visual_versions, linux_gcc_versio
         linux_gcc_versions = []
     if not osx_clang_versions:
         osx_clang_versions = []
+    if not gitlab_gcc_versions:
+        gitlab_gcc_versions = []
```

Comment:
It's true! Also, linux_clang_versions is absent. Now is fixed. Thanks for the review.